### PR TITLE
Fixing indent in 2 files 

### PR DESCRIPTION
**Pull request by BlockLucas**

### CODE REVIEW

File: tests/asyncapi-2.0/Message Object/valid.yaml [hunk 1]
```diff
@@ -42,8 +42,8 @@ channels:
             email:
               type: string
               format: email
-        examples:
-          - one:
-              email: hello@foo.bar
-          - two:
-              email: bye@foo.bar
+          examples:
```
Comment (derberg) [inline tests/asyncapi-2.0/Message Object/valid.yaml:45]: but this change will make `examples` part of `payload` object, but should be `message` -> https://www.asyncapi.com/docs/specifications/2.0.0/#a-name-messageobject-a-message-object

Comment (postatum) [reply tests/asyncapi-2.0/Message Object/valid.yaml:45]: Hm, right. I guess I got confused by the fact that the Schema Object may have examples as well.  

@BlockLucas, is there any particular reason why you made this change?

Comment (BlockLucas) [reply tests/asyncapi-2.0/Message Object/valid.yaml:45]: You are right, I got confused in the same way, will revert this file =)

**Comment by BlockLucas** [comment]
Yes, you are right, thanks for everything!

**Pull request closed without merging**